kolibri/plugins/device_management/assets/src/userSearchUtils.js

```diff
@@ -8,5 +6,8 @@ export function userMatchesFilter(user, searchFilter) {
 }
 
 export function filterAndSortUsers(users, pred) {
-  return orderBy(users.filter(pred), [user => user.username.toUpperCase()], ['asc']);
+  return users.filter(pred).sort(
+    // use 'search' option to ignore case rather than use locale defaults
+    (user1, user2) => user1.localeCompare(user2, null, { usage: 'search' })
```

Comment:
How does this function work? wouldn't `user1` and `user2` need to be strings? Before they were `FacilityUser` objects.

Reply:
also, this is the device_management version of this utility function. maybe that's why i haven't seen any problems on the facility_management app.

Reply:
You are right. I have no idea how/why this is working.